### ISSUE TITLE
Plotly js graph doesn't show any content when trying to have it show only the last items (sliding live update graph)

### ISSUE THREAD

**Issue by Six-wars** [plotly/plotly.js#4411]
I have the code below that loads a plotlyjs graph with content, I've got two graphs on the page with different id's. The issue with this is it doesn't show the content when loaded, when you click "reset index" it shows the content, all of it. What I intended to have is a slider of the last 30 elements.

    $.get(`/get-stock-trading/${symbol}`, function(response) {
        let complete = response['complete'];
        counter = response['data'].length;
        console.log(counter);
        console.log(max_items);

        if (complete){
            let y = [];
            let x = [];

            for (const row of response['data']){
                y.push(row[1]);
                x.push(row[0]);
            }

            let t2 = {
                y: y,
                x: x,
                type: 'scatter'
            };
        
            let plot_data = [t2];
            
            Plotly.newPlot('activity-chart', plot_data);

            if (counter > max_items){
                Plotly.relayout('activity-chart',{
                    xaxis: {
                                range: [counter-max_items, counter]
                            }
                });                    
            }
        } else {
            
        }
    });

I did a screen record of this https://youtu.be/q5aK4WYqsLI

Basically the issue is it won't show the line graph, it works if I just leave it to show all the content.

**Comment by etpinard** [comment]
This one will be hard to debug without a fully reproducible example. I suspect is up with your implementation code. Maybe other plotly.js users have experienced similar issues before, so you may want to report your question over at https://community.plot.ly/c/api/plotly-js

Thank you.

**Comment by etpinard** [comment]
Closing due to lack of activity
